frontend/www/js/omegaup/components/arena/Runs.test.ts

```diff
@@ -59,4 +59,139 @@ describe('Runs.vue', () => {
+            country: 'xx',
+            guid: '122000',
+            language: 'java',
+            memory: 1933312,
+            penalty: 0,
+            run_id: 227,
+            runtime: 316,
+            score: 0,
+            status: 'ready',
+            submit_delay: 0,
+            time: new Date(expectedDateProblem1),
+            type: 'normal',
+            username: 'username',
+            verdict: 'WA',
+          },
+          {
+            alias: 'alias',
+            classname: '',
+            contest_score: 0,
+            country: 'xx',
+            guid: '121000',
+            language: 'java',
+            memory: 1933312,
+            penalty: 0,
+            run_id: 228,
+            runtime: 316,
+            score: 100,
+            status: 'ready',
+            submit_delay: 0,
+            time: new Date(expectedDateProblem2),
+            type: 'normal',
+            username: 'username',
+            verdict: 'AC',
+          },
+          {
+            alias: 'alias',
+            classname: '',
+            contest_score: 0,
+            country: 'xx',
+            guid: '120500',
+            language: 'java',
+            memory: 1933312,
+            penalty: 0,
+            run_id: 229,
+            runtime: 316,
+            score: 100,
+            status: 'ready',
+            submit_delay: 0,
+            time: new Date(expectedDateProblem3),
+            type: 'normal',
+            username: 'username',
+            verdict: 'AC',
+          },
+          {
+            alias: 'alias',
+            classname: '',
+            contest_score: 0,
+            country: 'xx',
+            guid: '120000',
+            language: 'java',
+            memory: 1933312,
+            penalty: 0,
+            run_id: 230,
+            runtime: 316,
+            score: 100,
+            status: 'ready',
+            submit_delay: 0,
+            time: new Date(expectedDateProblem4),
+            type: 'normal',
+            username: 'username',
+            verdict: 'AC',
+          },
+          {
+            alias: 'alias',
+            classname: '',
+            contest_score: 0,
+            country: 'xx',
+            guid: '121500',
+            language: 'java',
+            memory: 1933312,
+            penalty: 0,
+            run_id: 231,
+            runtime: 316,
+            score: 100,
+            status: 'ready',
+            submit_delay: 0,
+            time: new Date(expectedDateProblem5),
+            type: 'normal',
+            username: 'username',
+            verdict: 'AC',
+          },
+        ],
+        showContest: true,
+        showDetails: true,
+        showDisqualify: true,
+        showPager: true,
+        showPoints: false,
+        showProblem: true,
+        showRejudge: true,
+        showUser: true,
+        username: null,
+      },
+    });
+
+    expect(
+      wrapper.find('tbody').findAll('td[name="guid"]').at(0).text(),
+    ).toContain('122000');
```

Comment:
```suggestion
    ).toEqual('122000');
```

aquí sabemos el contenido completo del texto, entonces podemos usar un matcher más específico.